crates/web-sys/src/features/gen_PublicKeyCredential.rs

```diff
@@ -37,11 +37,34 @@ extern "C" {
     pub fn get_client_extension_results(
         this: &PublicKeyCredential,
     ) -> AuthenticationExtensionsClientOutputs;
+    #[cfg(web_sys_unstable_apis)]
+    # [wasm_bindgen (static_method_of = PublicKeyCredential , js_class = "PublicKeyCredential" , js_name = isConditionalMediationAvailable)]
+    #[doc = "The `isConditionalMediationAvailable()` method."]
+    #[doc = ""]
+    #[doc = "[MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/API/PublicKeyCredential/isConditionalMediationAvailable)"]
+    #[doc = ""]
+    #[doc = "*This API requires the following crate features to be activated: `PublicKeyCredential`*"]
+    #[doc = ""]
+    #[doc = "*This API is unstable and requires `--cfg=web_sys_unstable_apis` to be activated, as"]
+    #[doc = "[described in the `wasm-bindgen` guide](https://rustwasm.github.io/docs/wasm-bindgen/web-sys/unstable-apis.html)*"]
+    pub fn is_conditional_mediation_available() -> ::js_sys::Promise;
     # [wasm_bindgen (static_method_of = PublicKeyCredential , js_class = "PublicKeyCredential" , js_name = isUserVerifyingPlatformAuthenticatorAvailable)]
     #[doc = "The `isUserVerifyingPlatformAuthenticatorAvailable()` method."]
     #[doc = ""]
     #[doc = "[MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/API/PublicKeyCredential/isUserVerifyingPlatformAuthenticatorAvailable)"]
     #[doc = ""]
     #[doc = "*This API requires the following crate features to be activated: `PublicKeyCredential`*"]
     pub fn is_user_verifying_platform_authenticator_available() -> ::js_sys::Promise;
+    # [wasm_bindgen (static_method_of = PublicKeyCredential , js_class = "PublicKeyCredential" , js_name = isConditionalMediationAvailable)]
+    #[doc = "The `isConditionalMediationAvailable()` method."]
+    #[doc = ""]
+    #[doc = "[MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/API/PublicKeyCredential/isConditionalMediationAvailable)"]
+    #[doc = ""]
+    #[doc = "*This API requires the following crate features to be activated: `PublicKeyCredential`*"]
+    pub fn is_conditional_mediation_available() -> ::js_sys::Promise;
+    # [wasm_bindgen (static_method_of = PublicKeyCredential , js_class = "PublicKeyCredential" , js_name = isExternalCTAP2SecurityKeySupported, catch)]
+    #[doc = "The `isExternalCTAP2SecurityKeySupported()` method."]
+    #[doc = ""]
+    #[doc = "*This API requires the following crate features to be activated: `PublicKeyCredential`*"]
+    pub fn is_external_ctap2_securitykey_supported() -> Result<::js_sys::Promise, JsValue>;
```

Comment:
The point of the whole WebIDL generation is so you don't write it by hand. Adding to this interface manually is something we don't support.

Your WebIDL for `isConditionalMediationAvailable` works just fine and was added by the generator just above. If you want to add `isExternalCTAP2SecurityKeySupported`, you should add it via WebIDL as well.

Reply:
I'm not going to add it? I thought I removed it from the PR.

Reply:
You should follow [this guide](https://github.com/rustwasm/wasm-bindgen/blob/main/guide/src/contributing/web-sys/supporting-more-web-apis.md) to add the APIs. Currently you have modified the `crates/web-sys/src/features/gen_PublicKeyCredential.rs` file by hand; you shouldn't touch these files at all.

Just rerunning the WebIDL generator should fix the issue.

Reply:
I don't understand what you mean here. I think this is getting too hard for me, and it's consuming a lot of my time to make little progress sorry.